okhttp-urlconnection/src/main/java/okhttp3/internal/huc/DelegatingHttpsURLConnection.java

```diff
@@ -147,7 +147,7 @@
     return delegate.getContentLength();
   }
 
-  @IgnoreJRERequirement // Should only be invoked on Java 7+.
+  @IgnoreJRERequirement // Should only be invoked on Java 7+ or Android API 24+.
```

Comment:
Remove Java 7 mention from these?

Reply:
Changed to _Should only be invoked on Java 8+ or Android API 24+_

Reply:
I wouldn't even mention 8 because it's the baseline. You're only ignoring for Android's sake.